app/helpers/application_helper/button/host_feature_button.rb

```diff
@@ -4,11 +4,6 @@ def visible?
     # TODO: @feature.nil? || @record.nil? || @record.supports?(@feature)
     # TODO: ensure start stop are supported in openstack
     unless @feature.nil? || @record.nil?
-      if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
-        return true if %w[start stop].include?(@feature.to_s)
-        return false
-      end
-
       return @record.supports?(@feature) if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::Host)
```

Comment:
Think we should get rid of this `if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::Host)` as well

Reply:
Actually does this just become a GenericFeatureButton at that point?

Reply:
We can't do that at the moment since `::Openstack::InfraManager` is also being called here and that would break the page i.e 
```
Error caught: [ActionView::Template::Error] undefined method `supports_standby?' for #<ManageIQ::Providers::Openstack::InfraManager:0x00007fe32e12f318>
```

Reply:
:+1: that's fine we need to fix that anyway